inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -247,6 +247,8 @@ StatusCode CNNNetworkNGraphImpl::addOutput(const std::string& layerName, size_t
                 }
                 if (_outputData.count(outputName) == 0) {
                     reshape();
+                }
+                if (_outputData.count(outputName) == 0) {
```

Comment:
Why not to add `addOutput` call to previous `if`? Now it looks like code duplication

Reply:
The original code calls `addOutput` and `reshape` in one `if`. I also don't fully understand how this additional check allows to solve the bug.
@GlebKazantaev Could you explain?

Reply:
Discussed offline with @ilyachur. Decided to remove addOutput call.